tutorial2_quantities_and_amounts/README.md

```diff
@@ -0,0 +1,59 @@
+# Tutorial 2
+
+The goals for this tutorial are:
+
+- to get comfortable with the [Github workflow](../GIT_SETUP.md), and with using their [local development environment](../README.md).
+- to understand how to load in a dataset using [d3-fetch](https://github.com/d3/d3-fetch) (d3.json, d3.csv, etc.).
+- to introduce [HTML svg](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/svg) coordinate system and how [d3-scales](https://bost.ocks.org/mike/selection/) can place graphical primitives on the page.
+- to practice creating a simple bar chart.
```

Comment:
i kinda want to remove 'simple' so that it sounds like a bigger thing. Maybe something like "learn the tools to create your own bar chart"?